Content.Server/Toilet/ToiletSystem.cs

```diff
@@ -122,15 +124,28 @@ private void OnInteractHand(EntityUid uid, ToiletComponent component, InteractHa
                 }
             }
 
-            // just want to up/down seat?
-            // check that nobody seats on seat right now
-            if (TryComp<StrapComponent>(uid, out var strap) && strap.BuckledEntities.Count != 0)
+            if (!CanToggle(uid))
                 return;
 
-            ToggleToiletSeat(uid, component);
             args.Handled = true;
         }
 
+        private void OnToggleSeatVerb(EntityUid uid, ToiletComponent component, GetVerbsEvent<AlternativeVerb> args)
+        {
+            if (!args.CanInteract || !args.CanAccess || !CanToggle(uid))
+                return;
+
+            string alterToiletSeatText = component.IsSeatUp ? Loc.GetString("toilet-seat-close") : Loc.GetString("toilet-seat-open");
+
+            var verb = new AlternativeVerb()
+            {
+                Act = () => { ToggleToiletSeat(uid, component); },
```

Comment:
you should include the CanToggle check inside the verb callback since it can happen after the initial check for CanToggle.